packages/mdadm/mdadm.spec

```diff
@@ -0,0 +1,44 @@
+%global debug_package %{nil}
```

Comment:
You shouldn't need `%global debug_package %{nil}` at the top if everything's working correctly. We have that in some specs but that's only "fine" in cases where it truly isn't needed, for packages like `release` with no compiled code. Everywhere else it's just tech debt that we'll have to pay off when we start doing something useful with debuginfo.